docs/user/azure/limits.md

```diff
@@ -0,0 +1,124 @@
+
+[Azure Service Limits][service-limits]
+
+Below, we'll identify OpenShift cluster needs and how those impact some of those limits.
+
+## VNet
+
+Each cluster creates its own VNet. The default limit of VNets per regions is 1000 and will allow 1000 clusters. To 
+have more than 1000 clusters, you will need to increase this limit. The default installation creates a single VNet.
+
+## Network Interfaces
+
+The default installation creates 6 network interfaces. The default limit per region is 65536. Additional network interfaces are
+created for additional machines and load balancers created by cluster usage and deployed workloads.
+
+## Public and Private IP Addresses
+
+By default, the installer distributes control-plane and compute machines across [all availability zones within a region][availability-zones] to provision the cluster in a highly available configuration.
+Please see [this map][az-map] for a current region map with availability zone count.
+We recommend selecting regions with 3 or more availability zones.
+You can [provide an install-config](../overview.md#multiple-invocations) to [configure](customization.md) the installer to use specific zones to override the defaults.
+
+The installer creates an internal load balancer and two subnets within the VNet. There is a subnet for the control plane nodes and there is a subnet for the compute nodes. The internal load balancer balances off the compute nodes while an external load balancer with a public IP address balances out the control plane nodes.
+
+## Network Security Groups
+
+Each cluster creates network security groups for the control plane and for compute nodes. The default limit of 5000
+for new accounts allows for many clusters to be created. The network security groups which exist after the default 
+install are:
+
+1. controlplane
+   * This allows the control-plane to be reached on port 6443 from anywhere
+2. node
+   * This allows worker nodes to be reached from the internet on ports 80 and 443
+
+## Instance Limits
+
+By default, a cluster will create:
+
+* One Standard_D4s_v3 bootstrap machine (removed after install)
+* Three Standard_D4s_v3 master nodes.
+* Three Standard_D2s_v3 worker nodes.
+
+The specs for the VM sizes (Dsv3-series) are as follows:
+
+* Standard_D4s_v3
+   * 4 vCPU's, 16GiB ram
+   * IOPs / Throughput (Mbps): (cached) 8000 / 512
+   * IOPs / Throughput (Mbps): (uncached) 6400 / 1152
+   * NICs / Bandwidth (Mbps): 2 / 2000
+   * NICs / Bandwidth (Mbps): 2 / 1000
+   * 32 GiB temp storage (SSD)
+   * 8 data disks max
+
+* Standard_D2s_v3
+   * 2 vCPU's, 8GiB ram
+   * IOPs / Throughput (Mbps): (cached) 4000 / 256
+   * IOPs / Throughput (Mbps): (uncached) 3200 / 384
+   * NICs / Bandwidth (Mbps): 2 / 1000
+   * 16 GiB temp storage (SSD)
+   * 4 data disks max
+
+More details on VM sizes can be found [here][sizes-general].
+
+Currently, these instance type counts are within a new account's default limit.
```

Comment:
Azure only gives you 10 vCPUs for default subscription, might have increased.. but the default requirement of 22 vCPUs is not met by default limits.

Reply:
Updated (default is 20 now).